internal/cmd/commands/authmethodscmd/ldap_funcs.go

```diff
@@ -0,0 +1,440 @@
+			})
+		case insecureTlsFlagName:
+			f.BoolVar(&base.BoolVar{
+				Name:   insecureTlsFlagName,
+				Target: &c.flagInsecureTls,
+				Usage:  "Skip the LDAP server SSL certificate validation (optional) - insecure and use with caution.",
+			})
+		case discoverDnFlagName:
+			f.BoolVar(&base.BoolVar{
+				Name:   discoverDnFlagName,
+				Target: &c.flagDiscoverDn,
+				Usage:  "Use anon bind to discover the bind DN of a user (optional).",
+			})
+		case anonGroupSearchFlagName:
+			f.BoolVar(&base.BoolVar{
+				Name:   anonGroupSearchFlagName,
+				Target: &c.flagAnonGroupSearch,
+				Usage:  "Use anon bind when performing LDAP group searches (optional).",
+			})
+		case upnDomainFlagName:
+			f.StringVar(&base.StringVar{
+				Name:   upnDomainFlagName,
+				Target: &c.flagUpnDomain,
+				Usage:  "The userPrincipalDomain used to construct the UPN string for the authenticating user (optional).",
+			})
+		case userDnFlagName:
+			f.StringVar(&base.StringVar{
+				Name:   userDnFlagName,
+				Target: &c.flagUserDn,
+				Usage:  "The base DN under which to perform user search (optional).",
+			})
+		case userAttrFlagName:
+			f.StringVar(&base.StringVar{
+				Name:   userAttrFlagName,
+				Target: &c.flagUserAttr,
+				Usage:  "The attribute on user entry matching the username passed when authenticating (optional).",
+			})
+		case userFilterFlagName:
+			f.StringVar(&base.StringVar{
+				Name:   userFilterFlagName,
+				Target: &c.flagUserFilter,
+				Usage:  "A go template used to construct a LDAP user search filter (optional).",
+			})
+		case enableGroupsFlagName:
+			f.BoolVar(&base.BoolVar{
+				Name:   enableGroupsFlagName,
+				Target: &c.flagEnableGroups,
+				Usage:  "Find the authenticated user's groups during authentication (optional).",
+			})
+		case groupDnFlagName:
+			f.StringVar(&base.StringVar{
+				Name:   groupDnFlagName,
+				Target: &c.flagGroupDn,
+				Usage:  "The base DN under which to perform group search.",
+			})
+		case groupAttrFlagName:
+			f.StringVar(&base.StringVar{
+				Name:   groupAttrFlagName,
+				Target: &c.flagGroupAttr,
+				Usage:  "The attribute that enumerates a user's group membership from entries returned by a group search (optional).",
+			})
+		case groupFilterFlagName:
+			f.StringVar(&base.StringVar{
+				Name:   groupFilterFlagName,
+				Target: &c.flagGroupFilter,
+				Usage:  "A go template used to construct a LDAP group search filter (optional).",
+			})
+		case certificatesFlagName:
+			f.StringSliceVar(&base.StringSliceVar{
+				Name:   certificatesFlagName,
+				Target: &c.flagCertificates,
+				Usage:  "PEM-encoded X.509 CA certificate that can be used as trust anchors when connecting to an LDAP server (optional). May be specified multiple times.",
+			})
+		case clientCertificateFlagName:
+			f.StringVar(&base.StringVar{
+				Name:   clientCertificateFlagName,
+				Target: &c.flagClientCertificate,
+				Usage:  "A client certificate in ASN.1 DER form encoded as PEM (optional).",
```

Comment:
This phrasing is inconsistent with the above, maybe we can use the same phrasing for both? What do you think about
```suggestion
				Usage:  "PEM-encoded X.509 client certificate that can be used to authenticate against an LDAP server(optional).",
```

Reply:
ty!